src/main/java/Entities/ProductReviewCountComparator.java

```diff
@@ -1,22 +1,24 @@
 package Entities;
-import java.util.Comparator;
 
-public class ItemReviewCountComparator implements Comparator<Product> {
+/**
+ * a class that compares 2 Products with respect to their review count
+ */
+public class ProductReviewCountComparator implements ProductComparator {
     /**
      * Takes 2 Items and compares their review count using .getReviewCount()
      * Returns 1, -1, 0 depending on the results of the comparison
-     * Default saved comparison: Highest to Lowest (descending)
+     * Default saved comparison: Lowest to Highest (ascending)
      * @param item1 The first name of Entities.Item
      * @param item2 The second name of Entities.Item
```

Comment:
all of the comparators have been formatted to saving lowest to highest as default. previously review count and review star saved highest as default. changed it because i made a helper method and having all default saved comparison as lowest to highest means i only had to make one helper.